### ISSUE TITLE
Redis purge all is not working when WP is installed in separate folder

### ISSUE THREAD

**Issue by pySilver** [rtCamp/nginx-helper#129]
Hi!

It turns out that redis "purge all" feature won't work for installations where wordpress is installed in a separate folder:

```
/domain.com
-- /wp # it's where WP lives
-- wp-config.php
-- index.php
```

The reason is here:
https://github.com/rtCamp/nginx-helper/blob/master/redis-purger.php#L684

```php
if( is_network_admin() ) {
				delete_keys_by_wildcard( $prefix . '*' );
				$this->log( '* Purged Everything! * ' );
			} else { // Else purge only site specific cache
				$parse = wp_parse_url( get_site_url() );
				$parse['path'] = empty( $parse['path'] ) ? '/' : $parse['path'];
				delete_keys_by_wildcard( $prefix . $parse['scheme'] . 'GET' . $parse['host'] . $parse['path'] . '*' );
				$this->log( '* ' . get_site_url() . ' Purged! * ' );
			}
```

Quickfix would be to use:
```get_home_url``` instead of ```get_site_url```

I believe we should change that, since cache urls are constructed against `WP_HOME`, not `WP_SITEURL`

or you can just change the query, so it would not contain parsed `path`.

**Comment by chandra-patel** [comment]
Hi @pySilver 

> you can just change the query, so it would not contain parsed path.
Here, we need path in case of multisite with sub-directory.

So, get_home_url() function solved issue.

I've accepted your PR. Thanks for the solution. I'm closing this issue.

Regards,
Chandra

**Comment by chandra-patel** [comment]
Hello,

I've released new version 1.9.10. Please update the plugin. Thanks.

**Comment by wp-networks** [comment]
Hello, 

What about: 
1) wordpress installed in /wp and then '/wp' removed from HOME
2) setup subdomain multisite
3) then, as per @nacin https://core.trac.wordpress.org/ticket/19796#comment:78 remove '/wp' from primary blog SITEURL so that now both at like
SITEURL=https://example.com
HOME=https://example.com

(fun fact: with appropriate apache rewrite rules in place, WP files can be in /wp and then install WP at https://example.com/wp-admin/... and it will work --> and it will not include '/wp' value in SITEURL or HOME even during install)

...so, relying on SITEURL holding correct path to WP files may not always work... what about ABSPATH?
Cheers, Max
